README.md

```diff
@@ -265,6 +265,27 @@ replicator.start();
 
 Read more in [the replication docs](https://github.com/cloudant/sync-android/blob/master/doc/replication.md).
 
+#### Authentication
+
+Sync-android uses session cookies by default to authenticate with the server if
```

Comment:
You should clarify that this happens when the user provides the username and password in the URL